stock_by_warehouse/models/product_product.py

```diff
@@ -26,6 +32,7 @@ def _warehouses_stock_recompute_onchange(self):
             self.warehouses_stock_recompute = True
             return
         self.warehouses_stock = self._compute_get_quantity_warehouses_json()
+        self.warehouses_stock_locations = self.get_warehouses_stock_location()
```

Comment:
s/get_warehouses_stock_location/_compute_get_stock_location/g

Try to stick to the naming convention of the module.

Reply:
Done :smile: